.github/workflows/docusaurus.yml

```diff
@@ -47,11 +47,17 @@ jobs:
       - name: Build Website
         run: |
           ./build-site.sh
```

Comment:
help me move `build-site.sh` to `scripts/build-site.sh`, make sure you check if we use this script elsewhere.